### PR TITLE
Adding message alert

### PR DESCRIPTION
**Pull request by WOLFIEEEE**
As a Microsoft team auto joiner, I think a message alert would be more beneficial, Twilio is a paid third party software but I think its free trial is more than enough as it provides enough message in its free trial only, For testing, you have to create a free trial account on Twilio and update the information in config.json and you are ready to go. Tried this as it might help people cause if you are using the auto joiner, I don't think you are going to sit in front of that. Hope this helps .

Steps to create an account on twillio

1. Open the link [https://www.twilio.com/try-twilio](url)
2. Sign up or login if you have an account already 
3. After sign up you have to verify your mobile number and email 
4. Login again in Twilio 
5. open Console there you will see the ACCOUNT SID and Auth token Copy both of that and add that into config.json
6. Below that you will get an option to get a trial number click on that and you will receive a trial number from which the message will be sent .
7. Copy that number and add it to twillio_no in config.json 
8. Add your number in Your_no in config.json 
9. Add a nickname of your choice 

and you are ready to go.

### CODE REVIEW

File: autoJoin.py [hunk 1]
```diff
@@ -84,6 +95,16 @@ def check_and_join_meeting():
...
+    global check
+    global meeting_name
+    meeting_name=browser.title
+    check = 0
+    #sending message to my number 
+    client.messages.create(
+    to='"'+data['your_no']+'"', 
+    from_='"'+data['twillio_no']+'"',
+    body='"'+"Hello " +data['nickname']+" I  joined the meeting named ** "+ meeting_name+ "** at time ** " + format(datetime.now()) + "   Hope you are doing well" + '"')
+    #message recived from that number
```
Comment (atharva-lipare) [inline autoJoin.py:107]: Put this code in an if condition. To be run only if the user wants to use twilio

File: autoJoin.py [hunk 2]
```diff
@@ -84,6 +95,16 @@ def check_and_join_meeting():
...
     wait_and_find_element_by_xpath('//button[.="Join now"]').click()  # join meeting
+    global check
+    global meeting_name
+    meeting_name=browser.title
+    check = 0
+    #sending message to my number 
+    client.messages.create(
+    to='"'+data['your_no']+'"', 
+    from_='"'+data['twillio_no']+'"',
+    body='"'+"Hello " +data['nickname']+" I  joined the meeting named ** "+ meeting_name+ "** at time ** " + format(datetime.now()) + "   Hope you are doing well" + '"')
```
Comment (atharva-lipare) [inline autoJoin.py:106]: Use f-strings or .format() for string formatting to improve readablility

File: autoJoin.py [hunk 3]
```diff
@@ -109,7 +130,16 @@ def check_and_end_or_leave_or_join_meeting():
             else:
                 browser.execute_script("document.getElementById('roster-button').click()")
         if curParticipants <= minParticipants and curParticipants != 0:  # leaves meeting for given condition
+
+                #twillion message Sending 
+            client.messages.create(
+            to='"'+data['your_no']+'"', 
+            from_='"'+data['twillio_no']+'"',
+            body='"'+"Hello "+data['nickname']+ " I left  the meeting named ** "+ meeting_name+"** as it wasn't ended becuase the current peoples in meeting were less than  **  "+ curParticipants + "** at the time of **"+format(datetime.now())+'**"')
```
Comment (atharva-lipare) [inline autoJoin.py:138]: *because the current *people in meeting were less than  **  "+ ~~curParticipants~~ minParticipants

**Comment by atharva-lipare** [comment]
Cool.

Just some suggestions,

- Use f-strings or .format() for string formatting to improve readability.
- Add an entry in the config file like `"use_twilio": true`
- Put the twilio  code in an `if` condition so it would execute only for users who want twilio.

Rest is fine, make the above changes and I'll test it on my end and merge the PR

File: autoJoin.py (hunk 3 above)
Comment (atharva-lipare) [inline autoJoin.py:138]: *because the current *people in meeting were less than  **  "+ ~~curParticipants~~ minParticipants

Just fix these minor typos

Comment (WOLFIEEEE) [reply autoJoin.py]: added the condition you asked for and changed the formatting,  Have a look @atharva-lipare

**Comment by atharva-lipare** [comment]
Nice @WOLFIEEEE, I tested it on my end and it worked as expected after changing the twilio version to 6.60.0.

On Python 3.8 on using twilio 5.7.0, I get this error: `ImportError: cannot import name 'parse_qs' from 'cgi' (/usr/lib/python3.8/cgi.py)` and on using twilio's latest version 6.60.0, it works fine. And twilio 6.60.0 is supported from Python 2.7 so using it's latest version shouldn't be a problem anyway.

So just change the twilio version in the requirements file to 6.60.0 and then I'll merge it.

It would be great if you also include the steps for making the twilio account in the README prerequisites section as an Optional step

**Comment by WOLFIEEEE** [comment]
@atharva-lipare Fixed the version in requirment.txt and added instruction in Readme.md

**Pull request merged**